Cubical/Categories/Displayed/Instances/Sets/Properties.agda

```diff
@@ -38,3 +38,47 @@ AllCartesianOversSETᴰ {c = A} {A'} B' f .isCartesian {A''} B'' g gfᴰ =
 isFibrationSet : isFibration (SETᴰ ℓ ℓ')
 isFibrationSet dᴰ = CartesianOver→CartesianLift (SETᴰ _ _)
   (AllCartesianOversSETᴰ _ _)
+
+open import Cubical.Categories.Displayed.Fibration.More
+open import Cubical.Data.Unit.Base
+open import Cubical.Data.Unit.Properties
+
+-- TODO: huhhhhhhhhhhhhhhhhhhhhh
+-- This is definitionally equivalent to LocalRightAdjointᴰ ????
+hasFiberedTerminalSet : hasFibTerminal (SETᴰ ℓ ℓ' , isFibrationSet)
+hasFiberedTerminalSet dᴰ .vertexᴰ _ = Unit* , isSetUnit*
+hasFiberedTerminalSet dᴰ .elementᴰ = tt
+hasFiberedTerminalSet dᴰ .universalᴰ .equiv-proof _ =
+  uniqueExists (λ _ _ → tt*) (isPropUnit tt tt) (λ _ p q → isSetUnit tt tt p q) λ _ _ → funExt λ _ → funExt λ _ → refl
+
+---- TODO: I don't understand why I can't put `refl` here
+--foobar : ∀ dᴰ → hasFibTerminal (SETᴰ ℓ ℓ' , isFibrationSet) dᴰ ≡ hasFibTerminal' (SETᴰ ℓ ℓ' , isFibrationSet) dᴰ
+--foobar dᴰ = {!!}
+
+hasFiberedTerminalSet' : hasFibTerminal' (SETᴰ ℓ ℓ')
+hasFiberedTerminalSet' dᴰ .vertexᴰ _ = Unit* , isSetUnit*
+hasFiberedTerminalSet' dᴰ .elementᴰ = tt
+hasFiberedTerminalSet' dᴰ .universalᴰ .equiv-proof _ =
+  uniqueExists (λ _ _ → tt*) (isPropUnit tt tt) (λ _ p q → isSetUnit tt tt p q) λ _ _ → funExt λ _ → funExt λ _ → refl
+
+--= total (SETᴰ ℓ ℓ' , isFibrationSet)
+
+-- I'm sure this is already somewhere, but whatever,
+-- let's just rewrite a short version
+open import Cubical.Categories.Limits.Terminal.More
+open import Cubical.Categories.Presheaf.Representable
+open import Cubical.Data.Sigma.Properties
+open UniversalElement
+setHasTerm : Terminal' (SET ℓ)
+setHasTerm .vertex = Unit* , isSetUnit*
+setHasTerm .element = tt
+setHasTerm .universal X .equiv-proof y = uniqueExists (λ _ → tt*) (isPropUnit tt tt)
+  (λ _ p' q' → isSetUnit tt tt p' q') λ _ _ → funExt λ _ → isPropUnit* tt* tt*
+
+open import Cubical.Categories.Displayed.Limits.Terminal
+
+SETᴰHasTermᴰ : ∀{ℓ ℓ'} → Terminalᴰ (SETᴰ ℓ ℓ') setHasTerm
```

Comment:
Closest I could find is `Cubical.Categories.Instances.Sets` has a proof that `SET` is complete